test-network/configtx/configtx.yaml

```diff
@@ -324,3 +324,18 @@ Profiles:
                 - *Org2
             Capabilities:
                 <<: *ApplicationCapabilities
+    TwoOrgsApplicationGenesis:
```

Comment:
Remove the old TwoOrgsOrdererGenesis and TwoOrgsChannel profiles.
Also grep for TwoOrgsOrdererGenesis, you'll find a long paragraph in network.sh explaining its use. This paragraph must also be updated to describe the new approach.

Reply:
I'm leaving the old paragraph for this next push just so we can compare when reviewing the edits I made and I'll remove it for the final push